docs/l2-adjacent-mode.rst

```diff
@@ -21,6 +21,12 @@ L2-adjacent mode is the **default mode of operation** for the |agent-short|.
    Many L3 segmentation mode parameters depend on other configuration parameters.
    Read about the :ref:`F5 agent configuration parameters <agent-config-parameters>` before changing these settings to ensure they don't conflict.
 
+**L2 Population Service**
+
+The OpenStack Neutron ML2 L2 population service is supported by the F5 LBaaS agent. When you enable L2 population, the agent registers for Neutron L2 population updates and populates tunnel FDB entries in your BIG-IP. When VIPs are placed on tenant overlay networks, the F5 LBaaS agent sends tunnel update messages to Open vSwitch agents informing them of TMOS device VTEPs. This enables tenant guest virtual machines or network node services to interact with the TMOS provisioned VIPs across overlay networks. An F5 LBaaS agent's connected TMOS VTEP addresses are placed in the agent's configurations and reported to Neutron. Without L2 population enabled, pool member migration will not function properly. You should enable this service if you migrate pool members to different virtual machines without re-creating your load balancer configuration. To enable the L2 population service in the F5 LBaaS agent, edit the agent configuration file, ``/etc/neutron/services/f5/f5-openstack-agent.ini``, and set ``l2_population = True``.
+
+You can also configure the agent to create static ARP entries on the TMOS devices. This removes the need for sending ARP broadcast (flooding) across the tunnels to learn the location of pool members. You do not need to enable static ARP optional and is not required if you enable L2 population service for the agent. To enable the static ARP entries, edit the agent configuration file, ``/etc/neutron/services/f5/f5-openstack-agent.ini``, and set ``f5_populate_static_arp = True``. Enabling static ARP entries can only be used with BIG-IP versions 12.x and greater.
```

Comment:
"With L2 population enabled, the F5 agent can also create static ARP entries on the BIG-IP device(s). This eliminates the need for the BIG-IP device to use ARP broadcast (flooding) across tunnels to learn the location of pool members. "


```
.. note::

   You can set the F5 Agent to create Static ARP entries for BIG-IP devices running version 12.x or later. 
```

Reply:
Fixed.